packages/components/src/dashicon/index.js

```diff
@@ -22,7 +22,8 @@ export default class Dashicon extends Component {
 			this.props.icon !== nextProps.icon ||
 			this.props.size !== nextProps.size ||
 			this.props.className !== nextProps.className ||
-			this.props.ariaPressed !== nextProps.ariaPressed
+			this.props.ariaPressed !== nextProps.ariaPressed ||
+			this.props.style !== nextProps.style
```

Comment:
This file is generated by the [Dashicons](https://github.com/WordPress/dashicons) project and copied here verbatim. The files should not be manually revised without an upstream patch.